src/sentry/models/projectsdk.py

```diff
@@ -0,0 +1,43 @@
+from collections.abc import Sequence
+from enum import Enum
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_model,
+    sane_repr,
+)
+
+
+class EventType(Enum):
+    PROFILE = 0
+    PROFILE_CHUNK = 1
+
+    @classmethod
+    def as_choices(cls) -> Sequence[tuple[int, str]]:
+        return (
+            (cls.PROFILE.value, "profile"),
+            (cls.PROFILE_CHUNK.value, "profile_chunk"),
+        )
+
+
+@region_silo_model
+class ProjectSDK(Model):
+    __relocation_scope__ = RelocationScope.Organization
+
+    date_updated = models.DateTimeField(auto_now=True)
+
+    project = FlexibleForeignKey("sentry.Project", on_delete=models.CASCADE)
+    event_type = BoundedIntegerField(choices=EventType.as_choices())
+    sdk_name = models.CharField()
+    sdk_version = models.CharField()
+
+    class Meta:
+        indexes = (models.Index(fields=["project", "event_type"]),)
+        unique_together = (("project", "event_type", "sdk_name"),)
```

Comment:
It sounds like you want to store only the latest sdk for each `event_type` based on your description? Does this mean we should just be uniquing on `project, event_type`?

Reply:
There's nothing that prevents users from sending data from multiple sdks to the same project. Think of examples where they send data from javascript + python to the same project.

Because of this, I'm looking to track the latest version for each project + event type + sdk.

Reply:
Ok, so how do you determine which is the latest sdk from multiple sdks?

Do we have plans to trim this table over time when an sdk hasn't been seen for a while?